sdk/core/System.BinaryData/api/System.BinaryData.netstandard2.0.cs

```diff
@@ -0,0 +1,27 @@
+namespace System
+{
+    public partial class BinaryData
+    {
+        public BinaryData(byte[] data) { }
```

Comment:
The implementation allows a null _data_ argument. Include an arg null check in the implementation, or make this parameter nullable. (Same comment on static factory.)

Reply:
I don't think we want to allow null